src/views/states/list/components/StateRow.tsx

```diff
@@ -2,37 +2,75 @@ import React, { FC } from 'react';
 import { NodeNetworkStateModelGroupVersionKind } from 'src/console-models';
 
 import { ResourceLink, RowProps, TableData } from '@openshift-console/dynamic-plugin-sdk';
-import { List, ListItem } from '@patternfly/react-core';
+import { List, ListItem, Title } from '@patternfly/react-core';
+import { ExpandableRowContent, Tbody, Td, Tr } from '@patternfly/react-table';
 import { V1beta1NodeNetworkState } from '@types';
+import { useNMStateTranslation } from '@utils/hooks/useNMStateTranslation';
 
-const StateRow: FC<RowProps<V1beta1NodeNetworkState>> = ({ obj, activeColumnIDs }) => {
-  const typesCount = obj?.status?.currentState?.interfaces.reduce((acc, iface) => {
+import InterfacesTable from './InterfacesTable';
+
+const StateRow: FC<RowProps<V1beta1NodeNetworkState, { rowIndex: number }>> = ({
+  obj,
+  activeColumnIDs,
+  rowData: { rowIndex },
+}) => {
+  const [isExpanded, setIsExpanded] = React.useState(false);
```

Comment:
You can import `useState` in this file, then you don't need to use `React.useState`, just `useState` here :)